tests/src/JIT/Regression/JitBlue/GitHub_18144/GitHub_18144.cs

```diff
@@ -0,0 +1,35 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Runtime.CompilerServices;
+using System.Runtime.Intrinsics;
+using System.Runtime.Intrinsics.X86;
+
+public class GitHub_18144
+{
+    [MethodImpl(MethodImplOptions.NoInlining)]
+    static void DoThis() { }
```

Comment:
Might it be worthwhile to have this create a number of locals with a different bit pattern and then validate that, on return, the locals in Main are still correct?

Right now, the test isn't actually testing any functionality is correct.